WordPress/gutenberg
# Accessibility improvements for List View Part 2

## Description


I have made further improvements to block-editor/list-view and components/tree-grid.

## Testing Instructions





1. Open a Post or Page.
2. Insert some Paragraph Blocks and a Columns Block with a selected variation.
3. Open the List View.
4. Move through the list with Up and Down Arrow keys.
5. Once you arrive at the Columns Block, notice how the link now contains aria-expanded. It seems that simply having aria-expanded on the parent table row wasn't working so I added it to the selection link.
6. Pressing Left Arrow should collapse the section, the screen reader should announce collapsed.
7. When pressing Right Arrow, you should hear expanded instead of first being placed on the Options button.
8. Once expanded, you can press Right Arrow again to move to the Options button.

This should also be easy to test on gutenberg.run. Just enter this PR number to build.

## Screenshots 

## Types of changes





Improvement and breaking change

## Checklist:
